components/resources/brave_components_strings.grd

```diff
@@ -682,6 +682,7 @@
       <message name="IDS_BRAVE_UI_ADS_CURRENTLY_VIEWING" desc="">Maximum ads received per hour: </message>
       <message name="IDS_BRAVE_UI_ADS_HISTORY_FILTER_ALL" desc="">All</message>
       <message name="IDS_BRAVE_UI_ADS_HISTORY_FILTER_SAVED" desc="">Saved</message>
+      <message name="IDS_BRAVE_UI_ADS_HISTORY_NONE" desc="">No ads viewed in the last [[totalDays]] days.</message>
```

Comment:
I know other messages in here use `[[...]]]` tags, but for anything new we should use `<ph>` placeholder tags. 

```
No ads viewed in the last <ph name="TOTAL_DAYS">$1</ph> days.
```

It means that `getLocale("...", { totalDays: x })` won't work. A simple alternative is to just do a text replace, something like:

```ts
getLocale('adsHistoryNone').replace('$1', String(totalDays || 0))
```

Reply:
Changed, thanks